traffic_control/clients/python/trafficops/tosession.py

```diff
@@ -268,7 +268,7 @@ def get_api_capabilities(self, query_params=None):
 	def get_api_capabilities_by_id(self, id=None):
 		"""
 		Get an API-capability mapping by ID
-		:ref:`to-api-api_capabilities-id`
+		:ref:`to-api-v1-api_capabilities-id`
```

Comment:
You don't need to change these references, the Python client is being upgraded to use the v2.0 API by default - so these are actually accurate (pending update, of course).

Reply:
these are the deprecated routes that are left in the v1 documentation that is what I updated them

Reply:
I guess that's true, but I'm just gonna change it back. That's not really a super big deal though, if you want to change it I'll let it go.